src/h264-extradata.js

```diff
@@ -0,0 +1,213 @@
+(function() {
```

Comment:
I moved the `H264ExtraData` object into its own file instead of having it live inside of `h264-stream.js`